src/main/java/com/atlauncher/gui/card/InstanceCard.java

```diff
@@ -86,6 +86,7 @@ public class InstanceCard extends CollapsiblePanel implements RelocalizationList
     private final JButton serversButton = new JButton(GetText.tr("Servers"));
     private final JButton openWebsite = new JButton(GetText.tr("Open Website"));
     private final JButton openButton = new JButton(GetText.tr("Open Folder"));
+    private final JButton openResourceButton = new JButton(GetText.tr("Open Resources"));
```

Comment:
The bottom row of buttons I don't think can show any more at the default size.

Can you copy what was done for the Play buttons where it's a dropdownbutton with the default action being Open Folder and then have this new one in the drop-down like play offline button is